core/trino-main/src/main/java/io/trino/type/DateTimes.java

```diff
@@ -648,10 +648,13 @@ public static LongTimestampWithTimeZone longTimestampWithTimeZone(long precision
     {
```

Comment:
No tests?

Reply:
https://github.com/trinodb/trino/pull/17511

Reply:
Thank you! :)